src/base/bittorrent/session.h

```diff
@@ -59,6 +60,9 @@
 #include "sessionstatus.h"
 #include "torrentinfo.h"
 
+#if LIBTORRENT_VERSION_NUM >= 10200 // #if __has_include(<libtorrent/fwd.hpp>)
+#include <libtorrent/fwd.hpp>
```

Comment:
I added this header in late versions of 1.1.x as well actually

Reply:
or if it's OK to drop support for libtorrent 1.1.3 and earlier, it could be included unconditionally

Reply:
1.1.3? I find out the header was released for the first time in 1.1.8.